.dockerignore

```diff
@@ -2,7 +2,6 @@
 .dockerignore
 .git
 .gitignore
-.github
```

Comment:
I guess there's no risk of NON ignoring the entire `.github` folder from Docker context, but would have make sense to add `!.github/CODEOWNERS` instead?

Reply:
I went this route mostly because .github is relatively small and it's much simpler to just copy the whole folder vs trying to pick out the individual file

Reply:
> I guess there's no risk of NON ignoring the entire `.github` folder from Docker context, but would have make sense to add `!.github/CODEOWNERS` instead?

PS: Same for the `COPY` stmt